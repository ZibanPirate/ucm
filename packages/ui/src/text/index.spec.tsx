import { render } from "@testing-library/react";

import { Text } from ".";

describe(`Testing component '${Text.name}' :`, () => {
  it(`should render with text 'Hi'`, () => {
    const { container } = render(<Text>OK</Text>);
    expect(container).toMatchSnapshot();
  });

  it(`should render a stretched '${Text.name}' component with text 'Used Cars Market'`, async () => {
    const { container } = render(<Text stretch>Used Cars Market</Text>);
    expect(container).toMatchSnapshot();
  });
});
