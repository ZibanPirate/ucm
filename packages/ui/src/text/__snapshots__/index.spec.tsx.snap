// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing component 'Text' : should render a stretched 'Text' component with text 'Used Cars Market' 1`] = `
<div>
  <span
    style="font-size: medium; margin: initial; display: flex; flex: 1;"
  >
    Used Cars Market
  </span>
</div>
`;

exports[`Testing component 'Text' : should render with text 'Hi' 1`] = `
<div>
  <span
    style="font-size: medium; margin: initial; display: flex;"
  >
    OK
  </span>
</div>
`;
